common/testing/testvars/test_vars.go

```diff
@@ -86,29 +88,29 @@ func (tv *TestVars) clone() *TestVars {
 	return tv2
 }
 
-func (tv *TestVars) cloneSet(typ, val string, key []string) *TestVars {
+func (tv *TestVars) cloneSet(typ string, key []string, val any) *TestVars {
```

Comment:
nit: it would be nice if key would be keys everywhere it's an array

Reply:
this is on purpose. I really don't see situation when more than 1 key is needed. So this is a way to use variadic func as func with optional parameter.